go.mod

```diff
@@ -22,17 +22,17 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/vincent-petithory/dataurl v1.0.0
 	go.uber.org/zap v1.24.0
-	golang.org/x/crypto v0.11.0
-	golang.org/x/mod v0.10.0
-	golang.org/x/sys v0.10.0
-	k8s.io/api v0.27.6
-	k8s.io/apimachinery v0.27.6
-	k8s.io/client-go v0.27.6
+	golang.org/x/crypto v0.14.0
+	golang.org/x/mod v0.13.0
+	golang.org/x/sys v0.13.0
+	k8s.io/api v0.28.0-alpha.0
+	k8s.io/apimachinery v0.28.0-alpha.0.0.20231013002001-194b6f8ddf62
+	k8s.io/client-go v0.28.0-alpha.0
```

Comment:
Shouldn't this point to `v0.27.z`?

Reply:
```
go get -u sigs.k8s.io/controller-runtime@release-0.15
go: upgraded github.com/emicklei/go-restful/v3 v3.10.2 => v3.11.0
go: upgraded github.com/evanphx/json-patch/v5 v5.6.0 => v5.7.0
go: upgraded github.com/go-openapi/jsonpointer v0.19.6 => v0.20.0
go: upgraded github.com/google/gnostic v0.6.9 => v0.7.0
go: added github.com/google/gnostic-models v0.6.9-0.20230804172637-c7be7c783f49
go: upgraded github.com/google/go-cmp v0.5.9 => v0.6.0
go: upgraded github.com/prometheus/client_golang v1.15.1 => v1.17.0
go: upgraded github.com/prometheus/client_model v0.4.0 => v0.5.0
go: upgraded github.com/prometheus/common v0.42.0 => v0.44.0
go: upgraded github.com/prometheus/procfs v0.9.0 => v0.12.0
go: upgraded go.uber.org/multierr v1.8.0 => v1.11.0
go: upgraded golang.org/x/crypto v0.11.0 => v0.14.0
go: upgraded golang.org/x/net v0.12.0 => v0.17.0
go: upgraded golang.org/x/oauth2 v0.10.0 => v0.13.0
go: upgraded golang.org/x/sys v0.10.0 => v0.13.0
go: upgraded golang.org/x/term v0.10.0 => v0.13.0
go: upgraded golang.org/x/text v0.11.0 => v0.13.0
go: upgraded gomodules.xyz/jsonpatch/v2 v2.3.0 => v2.4.0
go: upgraded k8s.io/api v0.27.6 => v0.28.2
go: upgraded k8s.io/apiextensions-apiserver v0.27.6 => v0.28.2
go: upgraded k8s.io/apimachinery v0.27.6 => v0.28.2
go: upgraded k8s.io/client-go v0.27.6 => v0.28.2
go: upgraded k8s.io/component-base v0.27.6 => v0.28.2
go: upgraded k8s.io/kube-openapi v0.0.0-20230928205116-a78145627833 => v0.0.0-20231010175941-2dd684a91f00
go: upgraded sigs.k8s.io/controller-runtime v0.15.2 => v0.15.3-0.20231013142619-9e09bc26e552
go: upgraded sigs.k8s.io/structured-merge-diff/v4 v4.2.3 => v4.3.0

[mansikulkarni@localhost windows-machine-config-operator]$ go get -u k8s.io/apimachinery@release-1.27
go: downgraded k8s.io/api v0.28.2 => v0.28.0-alpha.0
go: downgraded k8s.io/apiextensions-apiserver v0.28.2 => v0.28.0-alpha.0
go: downgraded k8s.io/apimachinery v0.28.2 => v0.28.0-alpha.0.0.20231013002001-194b6f8ddf62
go: downgraded k8s.io/client-go v0.28.2 => v0.28.0-alpha.0
go: downgraded k8s.io/component-base v0.28.2 => v0.28.0-alpha.0

```

Reply:
Looks like https://github.com/kubernetes/apimachinery/commit/194b6f8ddf62f55d1b8d997223e94ace2c1def31 is coming from the `release-1.27` branch, so I am guessing this okay 🤷🏽‍♂️ @sebsoto wdyt?